### PR TITLE
refactor agency show page solves #1622

### PR DESCRIPTION
**Pull request by StlMaris123**
In your PR did you:
This pr removes the conditional check on  if the jurisdiction is present since there is a similar method in the helper.
- Solves issue #1622

  - [x] Include a description of the changes?
  - [x] Mention the issue the PR addresses?
  - [ ] Include screenshots of any changes to the UI?
  - [ ] Isolate any changes to gems (meaning that any new, updated, or removed gems and resulting code changes should be in their own PR)?
  - [ ] Add and/or update specs for your code?

### CODE REVIEW

File: app/views/agencies/show.html.erb [hunk 1]
```diff
@@ -27,12 +24,6 @@
   <%= show_non_blank_fields "Email", @agency.email %>
 </p>
 
-<p>
-  <% if @agency.website.present? %>
-     Website: <%= link_to @agency.website, @agency.website %>
```
Comment (rlgreen91) [inline app/views/agencies/show.html.erb]: @StlMaris123 You haven't actually made the change to the helper, just deleted this section.

**Comment by StlMaris123** [comment]
Hey @rlgreen91 the `show_non_blank ` is already called on the agency show view

File: app/views/agencies/show.html.erb (hunk 1 above)
Comment (rlgreen91) [reply app/views/agencies/show.html.erb]: @StlMaris123 No, what I'm saying is that in line 13, you use the helper to display the jurisdiction if it's present.  I need you to do the same thing for the website - use the helper to display the website if it's present.  Replace lines 30-35 with that.

**Comment by rlgreen91** [comment]
@StlMaris123 Can you put it back in the order it was in before?  With website before notes?

**Pull request merged**